README.md

```diff
@@ -71,14 +72,14 @@ if (howMuchILikeSushi >= 3) {
 }
 
 if (x > 0) {
-  console.log("sushi is tasty");  
+  console.log("sushi is tasty");
 }
 ```
 
 Imagine that you take the code from this question, save it to a file called `food.js`, and run `node food.js` in your Terminal.
 
 What would be the output? Explain your answer.
-
+neither statement would show.  2 isn't greater than 3, so the delicious wouldn't appear.  x isn't defined, so that statement would appear.  It would be a default.
```

Comment:
I think you're giving the right answer - that this will cause the program to error out - but it's not clear from your explanation.